qualtricssurvey/tests/test_display.py

```diff
@@ -1,68 +1,71 @@
-import mock
+"""
+Test the Qualtrics Survey XBlock
+"""
 import unittest
 
+import mock
 from opaque_keys.edx.locations import SlashSeparatedCourseKey
 from xblock.field_data import DictFieldData
 
-from qualtricssurvey import QualtricsSurvey
-
+from qualtricssurvey.qualtricssurvey import QualtricsSurvey
 
-class QualtricsSurveyXblockTests(unittest.TestCase):
 
-    def make_an_xblock(self, **kw):
+def make_an_xblock(**kwargs):
+    """
+    Create and return an instance of the XBlock
+    """
+    course_id = SlashSeparatedCourseKey('foo', 'bar', 'baz')
+    runtime = mock.Mock(course_id=course_id)
+    scope_ids = mock.Mock()
+    field_data = DictFieldData(kwargs)
+    xblock = QualtricsSurvey(runtime, field_data, scope_ids)
+    xblock.xmodule_runtime = runtime
+    return xblock
 
-        course_id = SlashSeparatedCourseKey('foo', 'bar', 'baz')
-        runtime = mock.Mock(course_id=course_id)
-        scope_ids = mock.Mock()
 
-        field_data = DictFieldData(kw)
-        xblock = QualtricsSurvey(runtime, field_data, scope_ids)
-        xblock.xmodule_runtime = runtime
-        return xblock
+class QualtricsSurveyXblockTests(unittest.TestCase):
+    """
+    Test the XBlock
+    """
 
     def test_student_view(self):
         """
         Checks the student view with param_name but without
         anonymous_user_id.
         """
-
-        xblock = self.make_an_xblock()
+        xblock = make_an_xblock()
```

Comment:
This is being moved to be a top level function just to have it available globally during testing?

Reply:
No, it was a `pylint` error.
It never referenced `self` so it shouldn't be an instance method.

It's just a regular method.